clusters/build-clusters/arm01/assets/admin_cluster-admin-0_list.yaml

```diff
@@ -0,0 +1,123 @@
+kind: List
+apiVersion: v1
+items:
+- kind: Group
+  apiVersion: v1
+  metadata:
+    name: ci-admins
+  users:
+  - smarterclayton
+  - danmcp
+  - deads2k
+  - stevekuznetsov
+  - skuznets
+  - jwforres
+  - bbguimaraes
+  - droslean
+  - petr-muller
+  - coverprice
+  - derekwaynecarr
+  - sosiouxme
+  - tbielawa
+  - vikaslaad
+  - bparees
+  - mfojtik
+  - hongkailiu
+  - AlexNPavel
+  - jupierce
+  - bradmwilliams
+  - alvaroaleman
+  - mrunalp
+  - rphillips
+  - runcom
+  - haircommander
+  - alexanderConstantinescu
+  - umohnani8
+  - emilvberglind
+  - aravindh
```

Comment:
@LorbusChris why are you adding this person? 😄

Reply:
@aravindhp isn't that your RH kerberos user? (Logging in won't work with Githubs IdP for now)

Reply:
Ah yes, that is my RH kerberos user. Sorry for the noise.